README.md

```diff
@@ -22,7 +22,6 @@ Join [#events-app-beta](https://hubspotdev.slack.com/archives/C011GFF8KNZ) in th
 4. Add your [HubSpot API key](https://knowledge.hubspot.com/integrations/how-do-i-get-my-hubspot-api-key) for the portal by running `yarn hs secrets add APIKEY <api-key-goes-here>`. The API key is used by `eventSignup.js` to update the HubDB table
 5. Run `yarn start` which will build the javascript, auto-upload the files to your `defaultPortal`, and watch for changes
 6. Create a page in your portal that:
-    - Has the slug `/events/`
```

Comment:
We should update this to include instructions for adding the Upcoming Events module, too -- since you'll need to update that module to reflect the correct page.